azure-pipelines.yml

```
@@ -10,7 +10,7 @@ trigger:
   - main
   - release/*
 
-pool: NautilusBuild
+pool: Private Link Deployments (BUS)
```

Please revert this change as below:
```suggestion
pool: NautilusBuild
```

---

Reverted